nucleus/system/scei/cellos/lv2/sys_process.h

```diff
@@ -36,11 +36,19 @@ enum
     SYS_EVENT_FLAG_OBJECT           = 0x98,
 
     // Custom
-    SYS_PPU_THREAD_OBJECT           = 0x01,
+    SYS_PROCESS_OBJECT              = 0x01,
```

Comment:
The meaning behind the `// Custom` comment was just providing a "custom" identifier, given by us and not through reverse engineering of LV2. It was done in earlier days just as a temporary fix.

If you are sure that these objects types are in the LV2 kernel, feel free to move them with the rest above.

Again, you can commit this change yourself if you want. Otherwise I'll pull your changes and apply the fix myself.